src/main/java/ca/on/oicr/gsi/dimsum/util/reporting/reports/CaseTatReport.java

```diff
@@ -0,0 +1,206 @@
+package ca.on.oicr.gsi.dimsum.util.reporting.reports;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+import ca.on.oicr.gsi.cardea.data.Case;
+import ca.on.oicr.gsi.cardea.data.CaseDeliverable;
+import ca.on.oicr.gsi.cardea.data.CaseRelease;
+import ca.on.oicr.gsi.cardea.data.Project;
+import ca.on.oicr.gsi.cardea.data.Requisition;
+import ca.on.oicr.gsi.cardea.data.Sample;
+import ca.on.oicr.gsi.cardea.data.Test;
+import ca.on.oicr.gsi.dimsum.service.CaseService;
+import ca.on.oicr.gsi.dimsum.service.filtering.CaseFilter;
+import ca.on.oicr.gsi.dimsum.service.filtering.CaseFilterKey;
+import ca.on.oicr.gsi.dimsum.util.SampleUtils;
+import ca.on.oicr.gsi.dimsum.util.reporting.Column;
+import ca.on.oicr.gsi.dimsum.util.reporting.Report;
+import ca.on.oicr.gsi.dimsum.util.reporting.ReportSection;
+import ca.on.oicr.gsi.dimsum.util.reporting.ReportSection.TableReportSection;
+
+public class CaseTatReport extends Report {
+
+  private static class RowData {
+
+    private final Case kase;
+    private final Test test;
+
+    public RowData(Case kase, Test test) {
+      this.kase = kase;
+      this.test = test;
+    }
+
+    public Case getCase() {
+      return kase;
+    }
+
+    public Test getTest() {
+      return test;
+    }
+  }
+
+  private static final ReportSection<RowData> caseSection =
+      new TableReportSection<>("Case Report",
+          Arrays.asList(
+              Column.forString("Case ID", x -> x.getCase().getId()),
+              Column.forString("Projects",
+                  x -> getSortedProjectNameOrPipeline(x.getCase(), Project::getName)),
+              Column.forString("Pipeline",
+                  x -> getSortedProjectNameOrPipeline(x.getCase(), Project::getPipeline)),
+              Column.forString("Requisition", x -> x.getCase().getRequisition().getName()),
+              Column.forString("Assay", x -> x.getCase().getAssayName()),
+              Column.forString("Start Date",
+                  x -> x.getCase().getStartDate().format(DateTimeFormatter.ISO_LOCAL_DATE)),
+              Column.forString("Receipt Completed Date",
```

Comment:
"Receipt..." - "Full-Depth Completed" columns have "Date" at the end, but "Analysis Review..." and "Release Approval Completed" do not. These should be consistent. I think without the "Date" is better/more concise